tests/system/datastore/model_providers/test_huggingface.py

```diff
@@ -81,12 +84,18 @@ def test_basic_huggingface_model_runner(self, execution_mechanism):
         # The default Nuclio resource configuration is:
         # {"requests": {"cpu": "25m", "memory": "1Mi"}, "limits": {"cpu": "2", "memory": "20Gi"}}
         function.spec.resources = {
-            "limits": {"cpu": "5", "memory": "30Gi"},
-            "requests": {"cpu": "3", "memory": "1Mi"},
+            "limits": {"cpu": "7", "memory": "20Gi"},
+            "requests": {"cpu": "25m", "memory": "1Mi"},
         }
         function.spec.max_replicas = (
             1  # to avoid allocating extended resources to multiple pods
         )
+
+        # removed worker_timeout=500
+        # now removed gateway_timeout=600
+        function.with_http(worker_timeout=1000, workers=None)
```

Comment:
please explain why you are using `workers=None` and create a new bug for this issue. (add it's number here as well) .

I saw you are using it in few please explain it in all the relevant places.